### PR TITLE
Put entity injection for new questionnaire in a synchronized method

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

**Comment by bartcharbon** [comment]
test this please

**Comment by fdlk** [comment]
test this please

File: molgenis-questionnaires/src/main/java/org/molgenis/questionnaires/QuestionnairePluginController.java [hunk 1]
```diff
@@ -117,42 +126,47 @@ public String showThanks(@PathVariable("name") String name, Model model, HttpSer
...
 	}
 
-	public String getThankYouText(String questionnaireName)
+	private String getThankYouText(String questionnaireName)
 	{
 		return runAsSystem(() -> thankYouTextService.getThankYouText(questionnaireName));
 	}
 
-	public static class Questionnaire
+	private static class Questionnaire
```
Comment (fdlk) [inline molgenis-questionnaires/src/main/java/org/molgenis/questionnaires/QuestionnairePluginController.java]: Took me one hour to find this easter (halloween?) egg.
Private classes cannot be read in the FreeMarker template.
Please make public again.

Comment (Mark-de-Haan) [inline molgenis-questionnaires/src/main/java/org/molgenis/questionnaires/QuestionnairePluginController.java]: Ahh, the more you know :p

Comment (fdlk) [inline molgenis-questionnaires/src/main/java/org/molgenis/questionnaires/QuestionnairePluginController.java]: :D

**Pull request merged**